skyportal/models.py

```diff
@@ -641,4 +656,109 @@ class FollowupRequest(Base):
 
 User.followup_requests = relationship('FollowupRequest', back_populates='requester')
 
+
+class Thumbnail(Base):
+    # TODO delete file after deleting row
+    type = sa.Column(thumbnail_types, doc='Thumbnail type (e.g., ref, new, sub, dr8, ...)')
+    file_uri = sa.Column(sa.String(), nullable=True, index=False, unique=False)
+    public_url = sa.Column(sa.String(), nullable=True, index=False, unique=False)
+    origin = sa.Column(sa.String, nullable=True)
+    photometry_id = sa.Column(sa.ForeignKey('photometry.id', ondelete='CASCADE'),
+                              nullable=False, index=True)
+    photometry = relationship('Photometry', back_populates='thumbnails')
+    obj = relationship('Obj', back_populates='thumbnails', uselist=False,
+                       secondary='photometry',
+                       passive_deletes=True)
+
+
+class ObservingRun(Base):
+
+    instrument_id = sa.Column(
+        sa.ForeignKey('instruments.id', ondelete='CASCADE'), nullable=False
+    )
+    instrument = relationship(
+        'Instrument', cascade='save-update, merge, refresh-expire, expunge',
+        uselist=False, back_populates='observing_runs'
+    )
+
+    # name of the PI
+    pi = sa.Column(sa.Text)
+    observers = sa.Column(sa.Text)
+
+    sources = relationship(
+        'Obj', secondary='join(ClassicalAssignment, Obj)',
+        cascade='save-update, merge, refresh-expire, expunge',
+        passive_deletes=True
+    )
+
+    # let this be nullable to accommodate external groups' runs
+    group = relationship('Group', back_populates='observing_runs')
+    group_id = sa.Column(sa.ForeignKey('groups.id', ondelete='CASCADE'),
+                         nullable=True)
+
+    # the person who uploaded the run
+    owner = relationship('User', back_populates='observing_runs')
+    owner_id = sa.Column(sa.ForeignKey('users.id', ondelete='CASCADE'),
+                         nullable=False, index=True)
+
+    assignments = relationship('ClassicalAssignment', passive_deletes=True)
+    calendar_date = sa.Column(sa.Date, nullable=False, index=True)
+
+    @property
```

Comment:
probably makes sense to also make `property` which returns the moon position/phase at local midnight (which will help with planning).

Reply:
What other ephemeris data would be useful to return? Want to knock it all out at once.

Reply:
In PR #599 I have more ephemeris data but not moon. Will add that in a future PR